src/nvim/os/shell.c

```diff
@@ -63,6 +63,8 @@ char **shell_build_argv(const char *cmd, const char *extra_args)
 
   rv[i] = NULL;
```

Comment:
No, you're returning an array of strings. That NULL is marking the end of the array.